proto/filter.proto

```diff
@@ -97,7 +97,10 @@ enum ActionType {
     // Sets all layer minimum memory for any layer with the type "Render"
     SET_ALL_RENDER_LAYER_MEMORY = 7;
     // Sets all min cores for any layer with the type "Render"
-    SET_ALL_RENDER_LAYER_CORES = 8;
+    SET_ALL_RENDER_LAYER_CORES = 8 [deprecated = true];
+    SET_ALL_RENDER_LAYER_MIN_CORES = 10;
+    // Sets all max cores for any layer with the type "Render"
+    SET_ALL_RENDER_LAYER_MAX_CORES = 11;
```

Comment:
Let's maintain the ordering of field numbers, and add a comment to the deprecated field explaining what to use instead. Something like:

```
// Sets all min cores for any layer with the type "Render"
// This field is deprecated, use SET_ALL_RENDER_LAYER_MIN_CORES and
// SET_ALL_RENDER_LAYER_MAX_CORES instead.
SET_ALL_RENDER_LAYER_CORES = 8 [deprecated = true];
// Set memory optimizer
SET_MEMORY_OPTIMIZER = 9;
// Sets all min cores for any layer with the type "Render"
SET_ALL_RENDER_LAYER_MIN_CORES = 10;
// Sets all max cores for any layer with the type "Render"
SET_ALL_RENDER_LAYER_MAX_CORES = 11;
```

Reply:
Done! Removed the first line `// Sets all min cores for any layer with the type "Render"` since I wanted the main focus to be that it's deprecated